xrpl4j-model/src/test/java/org/xrpl/xrpl4j/model/client/fees/FeeUtilsTest.java

```diff
@@ -68,7 +68,8 @@ public void simpleComputeMultiSigFee() {
       .index(Hash256.of("A9C28A28B85CD533217F5C0A0C7767666B093FA58A0F2D80026FCC4CD932DDC7"))
       .build();
 
-    assertThat(computeMultisigNetworkFees(feeResult, object)).isEqualTo(XrpCurrencyAmount.ofDrops(60));
+    assertThat(computeMultisigNetworkFees(feeResult, object).recommendedFee())
```

Comment:
Let's assert the actual values for `low`, `medium` and `high` to ensure they are as expected, just to make sure we're wiring up values correctly.